test-cases/gutenberg/file.md

```diff
@@ -0,0 +1,219 @@
+-   While the upload is in progress tap on the File Block.
+
+**Expected Behavior**
+
+-   You should see the below message:
+![Upload stopped](../resources/local-file-upload-stop.png)
+
+**Next Step**
+
+-   Tap Yes
+
+**Expected Behavior**
+
+-   You should see that File block is cleared and the placeholder component is now visible.
+![Upload placeholder](../resources/local-file-upload-placeholder.png)
+
+--------------------------------------------------------------------------------
+##### TC005
+
+**Close the post with an ongoing file upload**
+
+**Steps**
+
+-   Add a File block and insert a file from your device.
+-   While there’s an ongoing upload, save the post as a draft.
+
+**Expected Behavior**
+
+-   Verify that you see the upload progress in post summary of the Post item in the Posts list.
+![File upload in post list](../resources/local-file-upload-post-list.png)
+
+**Next Steps**
+
+-   Wait until upload finishes.
+-   Re-open the post.
+
+**Expected Behavior**
+
+-   Verify that File block shows the file name.
+-   You can also switch to HTML mode to verify that the upload was complete. This is done by checking to see that the hyperlinks of the File block points to a remote media url and not a local one. 
+
+--------------------------------------------------------------------------------
+##### TC006
+
+**Replace the selected files**
+
+**Steps**
+
+-   Tap on an existing File Block that has a file. Note the file name for comparison in an upcoming step. 
+-   From the toolbar, click on the replace button. 
+-   The Choose file bottom sheet will now be visible. 
+-   Select a new file via the Choose from device or WordPress Media Library option. 
+
+**Expected Behavior**
+
+-   Verify that you see a new filename.
+
+--------------------------------------------------------------------------------
+##### TC007
+
+**Change the alignment in the File Block**
+
+**Step**
+- Press on the Alignment toolbar button.
+
+<kbd><img src="../resources/file-alignment.png" width="320"></kbd>
+
+- Choose one of the options. For example, Align Right.
+
+**Expected Behavior**
+
+- Check to ensure the UI represents the alignment accordingly.
+Note: Align left, Wide width and no option selected won't have visual differences.
+
+![Alignment](../resources/file-alignment-right.png)
+
+--------------------------------------------------------------------------------
+##### TC008
+
+The test cases below apply to both the Block Settings icon and the Link toolbar icon. The only differentiator is the Block Settings contain a "Show download button" option while the Link settings via the toolbar does not. 
+
+**The Link To Block Setting should open**
+
+**Steps**
+
+- Press the Block settings or Link Settings icon.
+- Press the Link to icon. 
+
+**Expected Behavior**
+
+- Two different options should appear: Media file and Attachment Page
+- Media File: The link should be set to the file itself.
+- Attachment page: The link should be set to the item page.
```

Comment:
I would add a step to check the preview site that the proper link was set and it is not broken.
Even though there's that known issue about recently uploaded files that will change their attachment url after autosaving on mobile.

Reply:
Sounds good!